docs/running-on-yarn.md

```diff
@@ -925,8 +925,14 @@ have configuration like:
 
 ```properties
   yarn.nodemanager.aux-services = spark_shuffle_x,spark_shuffle_y
-  yarn.nodemanager.aux-services.spark_shuffle_x.classpath = /path/to/spark-x-yarn-shuffle.jar,/path/to/spark-x-config
-  yarn.nodemanager.aux-services.spark_shuffle_y.classpath = /path/to/spark-y-yarn-shuffle.jar,/path/to/spark-y-config
+  yarn.nodemanager.aux-services.spark_shuffle_x.classpath = /path/to/spark-x-path/fat.jar:/path/to/spark-x-config
+  yarn.nodemanager.aux-services.spark_shuffle_y.classpath = /path/to/spark-y-path/fat.jar:/path/to/spark-y-config
+```
+Or
```

Comment:
It has to be colons right? This is just a fix rather than alternative syntax

Reply:
Yes, I have tested it in YARN 3.1.x. It has to be colons.

Reply:
So just change to colons? I think commas are a mistake, not an alternative

Reply:
Yes, I agree with you that commas are a mistake.

Reply:
Should I change this pr title

Reply:
Mostly I mean fix the PR, but yes you could do that too

Reply:
I may understand the alternative you talking. I have changed them both(commas -> colons). I offer two configs because I firstly deploy them not in fatjar but jars in the path.

Reply:
Ah I am misreading. The commas are still present in the first non-classpath argument. OK this makes sense.

Reply:
@srowen Thanks for your reviewing